client/src/components/worker/WorkerCard.js

```diff
@@ -7,19 +7,45 @@ import { Trades } from '../../shared/constants/worker/Trades';
 
 const WorkerCard = ({ worker }) => {
   const [isWorkerDetailOpen, setIsWorkerDetailOpen] = useState(false);
-
+  let workerImg
+  let workerType = worker.trades
+  switch (workerType) {
+    case "Electrician":
```

Comment:
we could probably store these in a constant file

by creating a new object that looks something like `client/src/shared/constants/worker/Trades.js`

```
export const Trades = {
  Electrician: 'Electrician',
  Mechanic: 'Mechanic',
  HVAC: 'Heating, Ventilation, and A/C',
  IT: 'Information Technology',
  Telecommunications: 'Telecommunications',
  Instrumentation: 'Instrumentation',
  Plumber: 'Plumber',
  Elevator: 'Elevator',
};
```

except for the values, they will be the image urls, and we can then map the url with `worker.trades` in line 48